server/server.js

```diff
@@ -1,3 +1,4 @@
+const newRelic = require('newrelic');
```

Comment:
Just FYI, if you are not using variable newRelic in server.js. Your can change your 1st line to `require('newrelic')`

Reply:
Oh nice, didn't realize that! I am going to remove it when testing my proxy server, so I will leave it for now. Good catch though!